pkg/selinux/selinux-client.js

```diff
@@ -108,8 +110,18 @@ export function init(statusChangedCallback) {
         // Example:
         // authlogin_nsswitch_use_ldap (on , on) Allow authlogin to nsswitch use ldap
         // Split by first ')', as the name cannot contain ')'
-        if (item)
-            result.push(item.split(")", 2)[1].trim());
+        if (item) {
+            const description = item.split(")", 2)[1].trim();
+            const state = item.split("(", 2)[1].startsWith("on") ? "yes" : "no";
+            const name = item.split(" ")[0];
+            const ansible = `
+- name: ${description}
+  seboolean:
+    name: ${name}
+    state: ${state}
+    persistent: yes\n`; // TODO persistent?
```

Comment:
It should be persistent, everything we do in Cockpit is -- we try to synchronize runtime and config state. Same as on the firewall or storage pages. IOW, this looks right, let's just drop the TODO.